<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="10,0">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>78</width>
          <height>59</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_running">
        <property name="text">
         <string>Running</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_rowcount">
        <property name="text">
         <string>Row count:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_colcount">
        <property name="text">
         <string>Col count:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="nextgenbut">
        <property name="text">
         <string>Next gen.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="launchbut">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopbut">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="output_current">
        <property name="text">
         <string>File output</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_time">
        <property name="text">
         <string>Time delay:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>78</width>
          <height>37</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addrowbut">
        <property name="text">
         <string>Add Row</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addcolbut">
        <property name="text">
         <string>Add Col</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removerowbut">
        <property name="text">
         <string>Remove Row</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removecolbut">
        <property name="text">
         <string>Remove Col</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>78</width>
          <height>56</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>nextgenbut</sender>
   <signal>pressed()</signal>
   <receiver>tableView</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
